docs/source/howto/installation_more/index.rst

```diff
@@ -62,3 +61,14 @@ Finally, check that the data directory has indeed changed::
 and try a simple AiiDA query with the new database.
 
 If everything went fine, you can delete the old database location.
+
+.. _how-to:installation:more:disable_repo_indexing:
+
+#############################################
+Exclude file repository from ``locate`` index
+#############################################
+
+Many Linux distributions include the ``locate`` command to quickly find files and folders, and run a daily cron job ``updatedb.mlocate`` to create the corresponding index.
+A large file repository can take a long time to index, up to the point where the hard drive is constantly indexing.
+
+In order to exclude the repository folder from indexing, add its path to the ``PRUNEPATH`` variable in the ``/etc/updatedb.conf`` configuration file (use ``sudo``).
```

Comment:
We can place this content within a dropdown close to the related note.

Reply:
Ok